core/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -115,12 +114,12 @@ public TachyonMaster(TachyonConf tachyonConf) {
         mTachyonConf.getInt(Constants.MASTER_MAX_WORKER_THREADS,
             Constants.DEFAULT_MASTER_MAX_WORKER_THREADS);
     Preconditions.checkArgument(mMaxWorkerThreads >= mMinWorkerThreads,
-            "tachyon.master.max.worker.threads can not "
+        "tachyon.master.max.worker.threads can not be "
             + "less than tachyon.master.min.worker.threads");
```

Comment:
Since there lines are already going to be split up, maybe use the longer constant values (ie. `Constants.MASTER_MAX_WORKER_THREADS`) instead of hardcoding them?